src/plugins/intel_gpu/src/kernel_selector/cl_kernels/rms_gpu_bfyx_opt.cl

```diff
@@ -38,7 +40,7 @@ KERNEL(rms_gpu_bfyx_opt)(
     const uint data_offset = data_idx * data_size;
     const uint subgroup_offset = get_sub_group_id() * get_sub_group_size() * items_num;
 
-    INPUT0_TYPE data[STACK_SIZE];
+    ACCUMULATOR_TYPE data[STACK_SIZE];
```

Comment:
Did you have a chance to check that this change doesn't affect kernel performance much in case of fp16 inputs? With this change registers usage becomes quite large, though it theoretically fits within limits, but still..